From 85b83aa734474b140955d952dc717e200e67d8b1 Mon Sep 17 00:00:00 2001
From: Brian C. Lane <bcl@redhat.com>
Date: Tue, 5 Apr 2011 11:25:44 -0700
Subject: [PATCH] Double buffer for fdasd_check_api_version (#693852)

The static buffer for the error string is too small. Double it.

Resolves: rhbz#693852
---
 libparted/labels/fdasd.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/libparted/labels/fdasd.c b/libparted/labels/fdasd.c
index 6d708f6..e235dd3 100644
--- a/libparted/labels/fdasd.c
+++ b/libparted/labels/fdasd.c
@@ -754,7 +754,7 @@ fdasd_check_api_version (fdasd_anchor_t *anc, int f)
 {
 	PDEBUG
 	int api;
-	char s[LINE_LENGTH];
+	char s[2*LINE_LENGTH];
 
         struct stat st;
         if (fstat (f, &st) == 0 && S_ISREG (st.st_mode)) {
-- 
1.7.4

